sleap/gui/learning/dialog.py

```diff
@@ -485,6 +504,32 @@ def get_every_head_config_data(
                 cfg = scopedkeydict.make_training_config_from_key_val_dict(
                     loaded_cfg_scoped
                 )
+
+                if len(self.labels.tracks) > 0:
+
+                    # for multiclass topdown the class vectors output stride
+                    # should be the max stride
+                    backbone_name = scopedkeydict.find_backbone_name_from_key_val_dict(
+                        tab_cfg_key_val_dict
+                    )
+                    max_stride = tab_cfg_key_val_dict[
+                        f"model.backbone.{backbone_name}.max_stride"
+                    ]
+
+                    # classes should be added here to prevent value error in
+                    # model since we don't add them in the training config yaml
```

Comment:
Nit: Should follow [Google style guide](https://google.github.io/styleguide/pyguide.html#385-block-and-inline-comments) (correct punctuation)